refactor/ui_backend/openapi.yaml

```diff
@@ -0,0 +1,77 @@
+openapi: 3.0.2
+info:
+  title: Covalent UI Backend Service API
+  version: 0.1.0
+paths:
+  /api/v0/ui/workflow/{dispatch_id}/task/{task_id}:
+    put:
+      tags:
+        - UI
+      summary: Update Ui
+      description: API Endpoint (/api/workflow/task) to update ui frontend
+      operationId: update_ui_api_v0_ui_workflow__dispatch_id__task__task_id__put
+      parameters:
+        - required: true
+          schema:
+            title: Dispatch Id
+            type: string
+          name: dispatch_id
+          in: path
+        - required: true
+          schema:
+            title: Task Id
+            type: integer
```

Comment:
task ID should be the same type it is in the runner API https://github.com/AgnostiqHQ/covalent/pull/149/files#diff-db80e5d84802cda9ffede916caa8a32c55476dd117106ff0f99189a7ea19d311R23